addons/binding/org.openhab.binding.dlinkupnpcamera/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,36 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: dlinkupnpcamera Binding
+Bundle-SymbolicName: org.openhab.binding.dlinkupnpcamera;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.1.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ClassPath: .
+Import-Package: 
+ com.google.common.collect,
+ org.apache.commons.lang,
+ org.eclipse.smarthome.config.core,
+ org.eclipse.smarthome.config.discovery,
+ org.eclipse.smarthome.config.discovery.inbox.events,
+ org.eclipse.smarthome.core.common.registry,
+ org.eclipse.smarthome.core.events,
+ org.eclipse.smarthome.core.items.events,
+ org.eclipse.smarthome.core.library.types,
+ org.eclipse.smarthome.core.thing,
+ org.eclipse.smarthome.core.thing.binding,
+ org.eclipse.smarthome.core.types,
+ org.eclipse.smarthome.io.net.http,
+ org.eclipse.smarthome.io.transport.upnp,
+ org.jupnp,
+ org.jupnp.model,
+ org.jupnp.model.meta,
+ org.jupnp.model.types,
+ org.openhab.io.net.actions,
+ org.osgi.service.component;version="1.2.2",
+ org.slf4j,
+ org.xml.sax,
```

Comment:
I can't see where any of the org.xml.sax/org.xml.sax.helpers packages are used, therefore please don't import them.